CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ Features
 - Update characteristics to use EIA 2020 RECS ([#1031](https://github.com/NREL/resstock/pull/1031))
 - Include HVAC secondary heating capabilities for project_testing ([#1090](https://github.com/NREL/resstock/pull/1090))
 - For heat pump upgrades, adds the ability to set the existing primary (non-shared) heating system as the backup system using only a single option from the lookup ([#1074](https://github.com/NREL/resstock/pull/1074))
+- options_saturations.csv is added to project_*/resources/ folder.
```

Comment:
Link to the PR?

Reply:
- oops, forgot the link. Okay to push that directly to develop or would you like a PR?

Reply:
Direct push.